client/src/App.js

```diff
@@ -32,8 +33,12 @@ const App = () => {
 				<Route path="/admin/cohorts" element={<Cohorts />} />
 				<Route path="/admin/cohorts/addcohort" element={<CohortForm />} />
 				<Route path="/admin/cohorts/:id" element={<Cohort />} />
+				<Route path="/admin/cohorts/updatecohort/:id" element={<UpdateCohort />} />
```

Comment:
I would go for path="/admin/cohorts/:id/edit/" so you have your entity id and then the action you are doing on it